ui/src/transaction.ts

```diff
@@ -359,7 +372,7 @@ export function emoji(tx: Tx): string {
   }
 }
 
-export function txIcon(tx: Tx): typeof SvelteComponent {
+export function txIcon(tx: Tx): typeof SvelteComponent | undefined {
```

Comment:
This is weird because `yarn typescript:check` complains on my branch if I don't explicitly add this:
```
ui/src/transaction.ts:375:33 - error TS2366: Function lacks ending return statement and return type does not include 'undefined'.

375 export function txIcon(tx: Tx): typeof SvelteComponent {
                                    ~~~~~~~~~~~~~~~~~~~~~~
```

On master this doesn't complain. The code that calls `txIcon` hasn't changed either.

Reply:
My question is whether we want to also provide an icon for a create org transaction. right now we don’t display anything.

Reply:
Ah, right. That also answers my question. It was just the exhaustiveness check that complained. Wish the error message was more descriptive.

3f5bf916984e960ef7003901fb37b931378611fc

Does the "orgs" icon make sense here @brandonhaslegs?
<img width="1009" alt="Screenshot 2021-06-15 at 12 29 52" src="https://user-images.githubusercontent.com/158411/122038355-cd3d3200-cdd5-11eb-9a10-bafd95318b72.png">

We're also using the same icon for the quorum:
<img width="476" alt="Screenshot 2021-06-15 at 12 33 00" src="https://user-images.githubusercontent.com/158411/122038447-e47c1f80-cdd5-11eb-9cfe-37b69e7ff82d.png">